www/themes/borg/css/components-nodes.css

```diff
@@ -115,6 +115,17 @@ label.github-clone-label {
     margin-left: 0;
   }
 }
+.node-type-project-module .block-field-node-field-download-count .field-item,
```

Comment:
All these CSS changes are going to be (accidentally) reverted on the next update of the borg theme if they aren't also repeated in https://github.com/backdrop-contrib/borg. 

If they are specific to this site only, they should be moved into the `backdropcms` theme instead.